### ISSUE TITLE
Python 3.11 support?

### ISSUE THREAD

**Issue by liudonghua123** [ergrelet/Scylla#3]
Hi, I tried to install `unlicense` via `pip install git+https://github.com/ergrelet/unlicense.git`, but I got `ERROR: Could not find a version that satisfies the requirement pyscylla<0.12.0,>=0.11.0 (from unlicense) (from versions: none)` errors.

```
C:\Users\Liu.D.H>pip install git+https://github.com/ergrelet/unlicense.git
Collecting git+https://github.com/ergrelet/unlicense.git
  Cloning https://github.com/ergrelet/unlicense.git to c:\users\liu.d.h\appdata\local\temp\pip-req-build-3jwxuigh
  Running command git clone --filter=blob:none --quiet https://github.com/ergrelet/unlicense.git 'C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-3jwxuigh'
  Resolved https://github.com/ergrelet/unlicense.git to commit 217d0d66c57b5deae45f8abd05c851a0bb3b0e6e
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting capstone<5.0,>=4.0 (from unlicense==0.3.0)
  Downloading capstone-4.0.2-py2.py3-none-win_amd64.whl (896 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 896.4/896.4 kB 40.6 kB/s eta 0:00:00
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='pypi.org', port=443): Read timed out. (read timeout=15)")': /simple/fire/
Collecting fire<0.5,>=0.4 (from unlicense==0.3.0)
  Downloading fire-0.4.0.tar.gz (87 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 87.7/87.7 kB 43.9 kB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting frida<16.0,>=15.1 (from unlicense==0.3.0)
  Downloading frida-15.2.2.tar.gz (11 kB)
  Preparing metadata (setup.py) ... done
Collecting lief>=0.12.2 (from unlicense==0.3.0)
  Downloading lief-0.13.2-cp311-cp311-win_amd64.whl (3.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 31.5 kB/s eta 0:00:00
INFO: pip is looking at multiple versions of unlicense to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement pyscylla<0.12.0,>=0.11.0 (from unlicense) (from versions: none)
ERROR: No matching distribution found for pyscylla<0.12.0,>=0.11.0

C:\Users\Liu.D.H>
```

I also tried to install `pyscylla` via `pip install git+https://github.com/ergrelet/Scylla.git`, but it failed again, see the following detailed logs.

<details><summary>Details</summary>

```
C:\Users\Liu.D.H>pip install git+https://github.com/ergrelet/Scylla.git
Collecting git+https://github.com/ergrelet/Scylla.git
  Cloning https://github.com/ergrelet/Scylla.git to c:\users\liu.d.h\appdata\local\temp\pip-req-build-9vnh2c32
  Running command git clone --filter=blob:none --quiet https://github.com/ergrelet/Scylla.git 'C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32'
  Resolved https://github.com/ergrelet/Scylla.git to commit be89e744841747786e510075d6d5f0fec331325e
  Running command git submodule update --init --recursive -q
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: pyscylla
  Building wheel for pyscylla (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for pyscylla (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [185 lines of output]
      running bdist_wheel
      running build
      running build_ext
      -- Building for: Visual Studio 17 2022
      -- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.22621.
      -- The C compiler identification is MSVC 19.35.32217.1
      -- The CXX compiler identification is MSVC 19.35.32217.1
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/cl.exe - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.35.32215/bin/Hostx64/x64/cl.exe - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- pybind11 v2.7.1
      -- Found PythonInterp: C:/Users/Liu.D.H/AppData/Local/Programs/Python/Python311/python.exe (found version "3.11.2")
      -- Found PythonLibs: C:/Users/Liu.D.H/AppData/Local/Programs/Python/Python311/libs/python311.lib
      -- Performing Test HAS_MSVC_GL_LTCG
      -- Performing Test HAS_MSVC_GL_LTCG - Success
      -- Configuring done (9.5s)
      -- Generating done (0.1s)
      -- Build files have been written to: C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/build/temp.win-amd64-cpython-311/Release
      MSBuild version 17.5.1+f6fdcf537 for .NET Framework

        Checking Build System
        Building Custom Rule C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/CMakeLists.txt
        decoder.c
        distorm.c
        instructions.c
        insts.c
        mnemonics.c
        operands.c
        prefix.c
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\thirdparty\distorm\src\prefix.c(1,1): warning C4819: The file contains a character that cannot be represented in the current code page (936). Save the file in Unicode format to prevent data loss [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\thirdparty\distorm.vcxproj]
        textdefs.c
        Generating Code...
        distorm.vcxproj -> C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\thirdparty\Release\distorm.lib
        Building Custom Rule C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/CMakeLists.txt
        tinystr.cpp
        tinyxml.cpp
        tinyxmlerror.cpp
        tinyxmlparser.cpp
        Generating Code...
        tinyxml.vcxproj -> C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\thirdparty\Release\tinyxml.lib
        Building Custom Rule C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/src/CMakeLists.txt
        AboutGui.cpp
        ApiReader.cpp
        Architecture.cpp
        Configuration.cpp
        ConfigurationHolder.cpp
        DeviceNameResolver.cpp
        DisassemblerGui.cpp
        DllInjection.cpp
        DllInjectionPlugin.cpp
        DonateGui.cpp
        DumpMemoryGui.cpp
        DumpSectionGui.cpp
        FunctionExport.cpp
        IATReferenceScan.cpp
        IATSearch.cpp
        ImportRebuilder.cpp
        ImportsHandling.cpp
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\ImportsHandling.cpp(447,27): warning C4477: 'swprintf_s' : format string '%d' requires an argument of type 'int', but variadic argument 2 has type 'unsigned __int64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\ImportsHandling.cpp(447,27): message : consider using '%zd' in the format string [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
        Logger.cpp
        MainGui.cpp
        NativeWinApi.cpp
        Generating Code...
        Compiling...
        OptionsGui.cpp
        PeParser.cpp
        PeRebuild.cpp
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\PeRebuild.cpp(368,48): warning C4311: 'type cast': pointer truncation from 'void *' to 'DWORD' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\PeRebuild.cpp(368,49): warning C4312: 'type cast': conversion from 'unsigned long' to 'void *' of greater size [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\PeRebuild.cpp(369,55): warning C4311: 'type cast': pointer truncation from 'void *' to 'DWORD' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\PeRebuild.cpp(369,56): warning C4312: 'type cast': conversion from 'unsigned long' to 'const void *' of greater size [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
        PickApiGui.cpp
        PickDllGui.cpp
        PluginLoader.cpp
        ProcessAccessHelp.cpp
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\ProcessAccessHelp.cpp(82,41): warning C4312: 'type cast': conversion from 'DWORD' to 'HANDLE' of greater size [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
        ProcessLister.cpp
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\ProcessLister.cpp(224,51): warning C4311: 'type cast': pointer truncation from 'HANDLE' to 'DWORD' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\ScyllaLib.vcxproj]
        Scylla.cpp
        StringConversion.cpp
        SystemInformation.cpp
        TreeImportExport.cpp
        Generating Code...
        ScyllaLib.vcxproj -> C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Release\ScyllaLib.lib
        Building Custom Rule C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/src/CMakeLists.txt
        main.cpp
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 1 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 2 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 3 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 4 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 5 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 6 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 7 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 8 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\src\main.cpp(150,50): warning C4477: 'swprintf_s' : format string '%p' requires an argument of type 'void *', but variadic argument 9 has type 'DWORD64' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\src\Scylla.vcxproj]
           Creating library C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/build/temp.win-amd64-cpython-311/Release/src/Release/Scylla.lib and object C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/build/temp.win-amd64-cpython-311/Release/src/Release/Scylla.exp
        Scylla.vcxproj -> C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\lib.win-amd64-cpython-311\Scylla.dll
        Building Custom Rule C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/bindings/python/CMakeLists.txt
        module.cpp
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\thirdparty\pybind11\include\pybind11\detail/type_caster_base.h(448,36): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\thirdparty\pybind11\include\pybind11\detail/type_caster_base.h(450,29): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\thirdparty\pybind11\include\pybind11\detail/type_caster_base.h(451,26): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2163,49): error C2039: 'frame': is not a member of '_ts' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\cpython/pystate.h(82,8): message : see declaration of '_ts' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2164): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2165,1): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2168,18): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2168,30): error C2027: use of undefined type '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\include\pytypedefs.h(22,16): message : see declaration of '_frame' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2167,33): error C2660: 'pybind11::detail::dict_getitem': function does not take 1 arguments [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\thirdparty\pybind11\include\pybind11\pytypes.h(513,19): message : see declaration of 'pybind11::detail::dict_getitem' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      C:/Users/Liu.D.H/AppData/Local/Temp/pip-req-build-9vnh2c32/thirdparty/pybind11/include\pybind11/pybind11.h(2167,33): message : while trying to match the argument list '()' [C:\Users\Liu.D.H\AppData\Local\Temp\pip-req-build-9vnh2c32\build\temp.win-amd64-cpython-311\Release\bindings\python\pyscylla.vcxproj]
      Traceback (most recent call last):
        File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 251, in build_wheel
          return _build_backend().build_wheel(wheel_directory, config_settings,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\build_meta.py", line 416, in build_wheel
          return self._build_with_temp_dir(['bdist_wheel'], '.whl',
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\build_meta.py", line 401, in _build_with_temp_dir
          self.run_setup()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 120, in <module>
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\__init__.py", line 107, in setup
          return distutils.core.setup(**attrs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
          dist.run_commands()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
          self.run_command(cmd)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\dist.py", line 1234, in run_command
          super().run_command(command)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\wheel\bdist_wheel.py", line 343, in run
          self.run_command("build")
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\dist.py", line 1234, in run_command
          super().run_command(command)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\command\build.py", line 131, in run
          self.run_command(cmd_name)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\dist.py", line 1234, in run_command
          super().run_command(command)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\command\build_ext.py", line 84, in run
          _build_ext.run(self)
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\command\build_ext.py", line 345, in run
          self.build_extensions()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\command\build_ext.py", line 467, in build_extensions
          self._build_extensions_serial()
        File "C:\Users\Liu.D.H\AppData\Local\Temp\pip-build-env-33d6iz5u\overlay\Lib\site-packages\setuptools\_distutils\command\build_ext.py", line 493, in _build_extensions_serial
          self.build_extension(ext)
        File "<string>", line 114, in build_extension
        File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 413, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--config', 'Release']' returned non-zero exit status 1.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyscylla
Failed to build pyscylla
ERROR: Could not build wheels for pyscylla, which is required to install pyproject.toml-based projects

[notice] A new release of pip is available: 23.1.2 -> 23.2
[notice] To update, run: python.exe -m pip install --upgrade pip

C:\Users\Liu.D.H>
```
</details> 

It would be nice if it build python wheels for 3.11. Maybe using [`cibuildwheel`](https://github.com/pypa/cibuildwheel) is a good choice.

**Comment by ergrelet** [comment]
Hi! Thanks for creating the issue, this has been on my TODO list for a while now. I'll get this fixed 👍

**Comment by ergrelet** [comment]
This is fixed with v0.11.1. Thanks for the suggestion by the way, `cibuildwheel` does the job perfectly 👍
